config/alienwhitelist.txt

```diff
@@ -24,6 +24,7 @@ singo - Gutter
 some~user - Species
 tastypred - Xenochimera
 verkister - Xenochimera
+WTFismyname - Xenomorph Hybrid
```

Comment:
These are ckeys not keys, therefore need to be all lowercase with no formatting.